packages/stores/src/ui-config/asset-info-config.ts

```diff
@@ -0,0 +1,152 @@
+import { PricePretty } from "@keplr-wallet/unit";
+import { action, autorun, computed, makeObservable, observable } from "mobx";
+import { TokenHistoricalPrice } from "src/queries-external/token-historical-chart/types";
+
+import { IPriceStore } from "../price";
+import {
+  PriceRange,
+  QueriesExternalStore,
+  TimeFrame,
+} from "../queries-external";
+
+const INITIAL_ZOOM = 1.05;
+const ZOOM_STEP = 0.05;
+
+export class ObservableAssetInfoConfig {
```

Comment:
Why is a ui-config needed?

Reply:
I think we decided on this in the TDD.

Though we still need to write a spec for it.

See comment for rationale

https://docs.google.com/document/d/1zw47fYdvLcEeJvRZiQqDfglSvpBLf0KDKyJ7Rw0UN3k/edit?disco=AAAA4kn39Mo